conformity/fields/__init__.py

```diff
@@ -47,3 +47,9 @@
     TimeDelta,
     TZInfo,
 )
+
+
+try:
+    from conformity.fields.country import CountryCodeField  # noqa
+except ImportError:
+    pass
```

Comment:
Why this addition here? It doesn't seem necessary to fix this bug, and we didn't do this for the Currency fields. IMO, it should be imported separately.

Reply:
Fixed.